src/Microsoft.Health.Fhir.Core/Messages/ConvertData/ConvertDataRequest.cs

```diff
@@ -58,8 +58,8 @@ public class ConvertDataRequest : IRequest<ConvertDataResponse>
 
         /// <summary>
         /// Reference for template collection.
-        /// The format is "<registryServer>/<imageName>:<imageTag>" for template collection stored in container registries.
-        /// Also supports image digest as reference. Will use 'latest' if no tag or digest present.
+        /// It can be the default templates "microsofthealth/fhirconverter:default", or an image on Azure Container Registry with digest "<registryServer>/<imageName>@<imageDigest>".
```

Comment:
Is `:default` a tag? Is this the only tag that is supported?

Reply:
Oh I think I get it. It must be exactly `microsofthealth/fhirconverter:default` or a reference with an image digest.